README.md

```diff
@@ -130,6 +130,8 @@ cd game-and-watch-retro-go
 
 # Place roms in ./roms/gb and ./roms/nes accordingly
 
+# for MacOS users, run make download_sdk
```

Comment:
This should not be needed. When running make through docker, `make` inside the container (from ubuntu) will download the sdk files.

Reply:
However, it doesn't hurt to have this for MacOS users.